webapp/packages/plugin-holidays/src/IHoliday.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+export interface IHoliday {
+  startDate: Date;
+  endDate: Date;
+  isHoliday: boolean;
+  name: string;
+  logoSrc: string;
+  iconSrc: string;
+  activeIconSrc?: string;
+  isCelebrating: boolean;
```

Comment:
Now I'm baffled; what is the difference between `isHoliday`?
It will be better to name it `animationActive` and actions like `startAnimation`, `stopAnimation`
(or maybe another word, but something to make some meaning difference with `isHoliday`